### PR TITLE
refactor: Added test for Area.java and refactor

### PR DESCRIPTION
**Pull request by amarildoaliaj**
### **Describe your change:**
- [x] Test + Refactoring

### **Checklist:**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] All new Java files are placed inside an existing directory.
- [x] All filenames are in all uppercase characters with no spaces or dashes.
- [x] All functions and variable names follow Java naming conventions.

### CODE REVIEW

File: src/main/java/com/thealgorithms/maths/Area.java [hunk 1]
```diff
@@ -61,83 +39,125 @@ private static double surfaceAreaCube(double sideLength) {
...
     }
 
     /**
-     * Calculate the area of a trapezium
+     * Calculate the area of a trapezium.
      *
-     * @param base1 upper base of trapezium
-     * @param base2 bottom base of trapezium
+     * @param base1  upper base of trapezium
+     * @param base2  bottom base of trapezium
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/maths/Area.java:134]:
```suggestion
     * @param base1 upper base of trapezium
     * @param base2 bottom base of trapezium
```
No need for extra space

File: src/main/java/com/thealgorithms/maths/Area.java [hunk 2]
```diff
@@ -61,83 +39,125 @@ private static double surfaceAreaCube(double sideLength) {
...
+        }
         return sideLength * sideLength;
     }
 
     /**
-     * Calculate the area of a triangle
+     * Calculate the area of a triangle.
      *
-     * @param base base of triangle
+     * @param base   base of triangle
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/maths/Area.java:99]:
```suggestion
     * @param base  base of triangle
```

File: src/main/java/com/thealgorithms/maths/Area.java [hunk 3]
```diff
@@ -61,83 +39,125 @@ private static double surfaceAreaCube(double sideLength) {
...
+        }
         return base * height / 2;
     }
 
     /**
-     * Calculate the area of a parallelogram
+     * Calculate the area of a parallelogram.
      *
-     * @param base base of parallelogram
+     * @param base   base of parallelogram
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/maths/Area.java:116]:
```suggestion
     * @param base  base of a parallelogram
```

File: src/main/java/com/thealgorithms/maths/Area.java [hunk 4]
```diff
@@ -61,83 +39,125 @@ private static double surfaceAreaCube(double sideLength) {
...
         return 4 * Math.PI * radius * radius;
     }
 
     /**
-     * Calculate the area of a rectangle
+     * Calculate the area of a rectangle.
      *
      * @param length length of rectangle
-     * @param width width of rectangle
+     * @param width  width of rectangle
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/maths/Area.java:53]:
```suggestion
     * @param width width of a rectangle
```

**Review by debasishbsws — APPROVED** [review]
Remove these extra spaces from the comments

File: src/main/java/com/thealgorithms/maths/Area.java (hunk 1 above)
Comment (amarildoaliaj) [reply src/main/java/com/thealgorithms/maths/Area.java:134]: Those whitespace are recommended by IntelliJ IDEA to put text on the same column. In fact, javadoc looks better

Comment (debasishbsws) [reply src/main/java/com/thealgorithms/maths/Area.java:134]: Oh, thanks for sharing that. Everything looks greate

**Review by debasishbsws — APPROVED** [review]
Fine by me

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Review by siriak — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Comment by amarildoaliaj** [comment]
Done

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**